freqtrade/optimize/lookahead_analysis_helpers.py

```diff
@@ -137,6 +137,16 @@ def calculate_config_overrides(config: Config):
                         'just to avoid false positives')
             config['dry_run_wallet'] = min_dry_run_wallet
 
+        if 'fee' not in config or config['fee'] != 0.02:
+            logger.info('fee was not set to a fixed value of 0.02. ')
+            config['fee'] = 0.02
```

Comment:
wow, a fee of 2% ... that's not only excessive - it'll change results on a TON of strategies ... almost all of which will be negative with that kind of fee (you start with ~-2% on every trade).

While we _can_ default to a specific value - we should accept whatever value a user supplies (currently, my explicit config would be thrown away).
If i use `fee=0.001` in the config because i know my fee level on the exchange will be that - then no command should decide "we use something different" - unless it's the live exchange telling you you're wrong :laughing: 



---

If i'm honest, i'd implement this differently. 
Let the "first" backtest run through - and use the fee value from there (`backtesting.fee`) to assign it to the config going forward, so all subsequent, "individual" backtests then use that first value.

Fees can be quite different depending on markets / exchanges - and forcing one specific value across all will accomplish little - and will potentially break tons of strategies that rely on a specific "low" fee.

Reply:
whoops

Reply:
that's correct - but 2% is ... excessive, maybe try 0.001 (0.1%) ... that shouldn't impact actual results too much.

I'd still keep a "user desire" though - as you say - as long as it's defined, we don't "really" care what it is.

Reply:
yes 2% is excessive, that was a brain fart

Reply:
the amount of fees really does not matter in this case
it just matters that the tests are reproducible and that the fees are static